src/util/inhibitors.ts

```diff
@@ -0,0 +1,15 @@
+import { Inhibitor } from 'cookiecord';
+import { trustedRoleId } from '../env';
+
+export const isTrustedMember: Inhibitor = async (msg, client) => {
+	if (!msg.guild || !msg.member || msg.channel.type !== 'text') {
+		return "You can't use that command here.";
```

Comment:
Make this message more similar to the rest? (lowercase and `:warning:`)

p.s: also L13